src/SourceBuild/content/repo-projects/sdk.proj

```diff
@@ -1,19 +1,10 @@
 <Project Sdk="Microsoft.Build.NoTargets">
 
   <PropertyGroup>
-    <LogVerbosityOptOut>true</LogVerbosityOptOut>
-
     <!-- Tests are only building on Windows - https://github.com/dotnet/source-build/issues/4882 -->
     <DotNetBuildTestsOptOut Condition="'$(OS)' != 'Windows_NT'">true</DotNetBuildTestsOptOut>
 
     <BuildArgs>$(BuildArgs) /p:PortableRid=$(PortableRid)</BuildArgs>
-    <BuildArgs>$(BuildArgs) $(FlagParameterPrefix)v $(LogVerbosity)</BuildArgs>
-
-    <!--
-      Setting NETCoreAppMaximumVersion to a high version so that the sdk doesn't complain if we're restoring/publishing for a higher version than the sdk.
-      See https://github.com/dotnet/sdk/issues/1512#issuecomment-377082883
```

Comment:
why do we no longer need this?

Reply:
We will rev to the next major version differently with the VMR for .NET 11. At that point I expect that we will globally flow a set of properties in to the inner repos which will allow to target the next major version. The property is currently not needed as maximum version is 10.